### PR TITLE
Cards

### PR DESCRIPTION
**Pull request by itsmebipul00**
Designed Cards of all types

### CODE REVIEW

**Review by iRohitGaur — APPROVED** [review]
Nice work! Good to merge 👍🏻

File: Components/Alerts/Alerts.css [hunk 1]
```diff
@@ -43,3 +39,11 @@ code {
 .code::-webkit-scrollbar-thumb {
   background: hsl(var(--clr-dark));
 }
+
+.alert-information span {
+  padding: 0.5rem;
```
Comment (zoef-yt) [inline Components/Alerts/Alerts.css:44]: px can be used here.

**Review by zoef-yt — APPROVED** [review]
Looks good. Good to go.

**Review by srishtigautam21 — APPROVED** [review]
understandable class naming the code looks clean....good to merge

**Review by bhushannaikaroskar — APPROVED** [review]

File: Components/Cards/Cards.css [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+  background: hsl(var(--clr-dark));
+}
+
+.close-icon-horizontal {
+  inset: 2% 0 0 94%;
+}
+
+.close-icon-verticle {
+  inset: 2% 0 0 88%;
+}
```
Comment (bhushannaikaroskar) [inline Components/Cards/Cards.css:135]: I think there is typo here it should be close-icon-vertical. Rest good to go

**Review by bhushannaikaroskar — APPROVED** [review]

**Review by varun13169 — APPROVED** [review]

**Comment by itsmebipul00** [comment]
Thankyou guys for the reviews!!

**Pull request merged**